workitem/workitemtype_repository.go

```diff
@@ -120,7 +120,24 @@ func ClearGlobalWorkItemTypeCache() {
 	cache.Clear()
 }
 
-// Create creates a new work item in the repository
+// CreateFromModel creates a new work item type in the repository without any
+// fancy stuff.
+func (r *GormWorkItemTypeRepository) CreateFromModel(ctx context.Context, model *WorkItemType) (*WorkItemType, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitemtype", "create_from_model"}, time.Now())
```

Comment:
we don't use underscore so let's keep it consistent `createfrommodel`. In other moment, we can change it or not to a diff format.